tasks/task109_SMSspamcollection_SpamSMSdetection.json

```diff
@@ -8,7 +8,7 @@
     "Source": [
         "SMS Spam Collection v.1 (https://www.dt.fee.unicamp.br/~tiago/smsspamcollection/)"
     ],
-    "Definition": "In this task, you're given Text messages which can be classified between two classes Spam or Ham messages based on the content of the text message. Spam messages are messages which contain text which is irrelevant to the receiver and the message can be equally applicable to many other potential recipients.",
+    "Definition": "In this task, you're given Text messages which can be classified between two classes, Spam or Ham messages, based on the content of the text message. Messages which contain text that is irrelevant to the receiver and it can be equally applicable to many other potential recipients than label it as 'spam', otherwise label it as 'ham'.",

```

Comment:
There are several grammatical issues here. Please run a grammar check here (e.g., "it" does match "Messages"; "than label" -> "must be labeled").